libraries/Microsoft.Bot.Builder.Dialogs/ObjectPath.cs

```diff
@@ -710,6 +710,11 @@ private static object GetNormalizedValue(object value, bool json)
                 if (value is JToken || value is JObject || value is JArray)
                 {
                     val = Clone((JToken)value);
+                    if (val.GetType() == typeof(JValue) && ((JToken)val).Type == JTokenType.Date)
+                    {
+                        var temp = val.ToString();
+                        val = DateTime.Parse(temp).ToString("yyyy-MM-ddTHH:mm:ss.fffZ");
+                    }
```

Comment:
Why not change addTime, addSecond() etc to take in DATE object?  I don' think this is a good idea.

Reply:
Yes. @cosmicshuai let's take this on the expression side. I think I answered your Q there but agree with Tom. We should do this on the prebuilt functions side.

Reply:
Sure, I will revert the changes.